.github/workflows/python-cross-platform-support.yml

```diff
@@ -18,13 +18,13 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v3
 
-      - name: Set up Python 3.9.13
+      - name: Set up Python 3.11
         uses: actions/setup-python@v4
         with:
-          python-version: 3.9.13
+          python-version: 3.11
 
-      - name: Install the AaC PUML Package
-        run: python -m pip install -e .
+      - name: Install the AaC Package
+        run: python -m pip install -e .[all]
```

Comment:
Still looks like something with cross platform didn't get pulled correctly cause I just did a PR getting rid of this in the templates?